src/physics/include/grins/physics_naming.h

```diff
@@ -37,10 +37,10 @@ namespace GRINS
   public:
 
     static void set_suffix( const std::string& suffix )
-    { _suffix = suffix; }
+    { _suffix() = suffix; }
```

Comment:
Let's be explicit: `PhysicsNaming::suffix()` (assuming the function name change).